astore/client/auth/auth.go

```diff
@@ -98,10 +104,39 @@ func (c *Client) Login(username, domain string, o LoginOptions) (string, error)
 		return "", fmt.Errorf("server returned invalid nonce, please try again - %s", err)
 	}
 
-	decrypted, ok := box.Open(nil, []byte(tres.Token), nonce.ToByte(), servPub.ToByte(), priv)
+	decrypted, ok := box.Open(nil, tres.Token, nonce.ToByte(), servPub.ToByte(), priv)
 	if !ok {
 		return "", fmt.Errorf("could not decrypt returned token")
 	}
 
-	return string(decrypted), nil
+	caPublicKey, _, _, _, err := ssh.ParseAuthorizedKey(tres.Capublickey)
```

Comment:
technically this doesn't help for testability since during testing we're just going to execute the cobra command, but implemented for readability at least